charts/argo-cd/README.md

```diff
@@ -427,6 +427,9 @@ NAME: my-release
 | configs.cm.annotations | object | `{}` | Annotations to be added to argocd-cm configmap |
 | configs.cm.create | bool | `true` | Create the argocd-cm configmap for [declarative setup] |
 | configs.cm.url | string | `""` | Argo CD's externally facing base URL (optional). Required when configuring SSO |
+| configs.cmp.annotations | object | `{}` | Annotations to be added to argocd-cmp-cm configmap |
+| configs.cmp.create | bool | `true` | Create the argocd-cmp-cm configmap |
```

Comment:
Do we want this by default or should we default to `false` and preserve previous behaviour?

Reply:
@mbevc1 Good catch. Maybe false should be default as majority of people will not need custom CMP plugin.

Reply:
:+1: